README.md

```diff
@@ -32,269 +34,51 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
-
-Since the container is running detached, you will not see any logs. However, you can follow the logs by running:
-
-```bash
-docker-compose logs -f
-```
-
-Use CTRL+C to stop monitoring the logs and return to a prompt.
-
-### Shutdown
-
-The deployment can be brought down by running:
-
-```bash
-docker-compose down
-```
-
-### Manage example data
-
-[Deephaven's examples repository](https://github.com/deephaven/examples) contains data sets that are useful when learning 
-to use Deephaven. These data sets are used extensively in Deephaven's documentation and are needed to run some examples. [Deephaven's examples repository](https://github.com/deephaven/examples) contains documentation on the available data sets and how to manage them. 
-
-If you have chosen a deployment with example data, the example data sets will be downloaded. Production deployments containing your own data will not need the example data sets.
-
-
-To upgrade a deployment to the latest example data, run:
-
-```bash
-docker-compose run examples download
-```
-
-To see what other example data management commands are available, run:
-
-```bash
-docker-compose run examples
-```
-
-If your deployment does not have example data, these commands will fail with `ERROR: No such service`.
-
-
-## Run Deephaven IDE
-
-Once Deephaven is running, you can launch a Deephaven IDE in your web browser.  Deephaven IDE allows you to interactively analyze data.
-
-- If Deephaven is running locally, navigate to [http://localhost:10000/ide/](http://localhost:10000/ide/).
-- If Deephaven is running remotely, navigate to `http://<hostname>:10000/ide/`, where `<hostname>` is the address of the machine Deephaven is running on.
-
-![alt_text](docs/images/ide_startup.png "Deephaven IDE")
-
-# First query
-
-From the Deephaven IDE, you can perform your first query.
-
-This script creates two small tables: one for employees and one for departments.
-It joins the two tables on the DeptID column to show the name of the department
-where each employee works.
-
-```python
-
-from deephaven.TableTools import newTable, stringCol, intCol
-from deephaven.conversion_utils import NULL_INT
-
-left = newTable(
-        stringCol("LastName", "Rafferty", "Jones", "Steiner", "Robins", "Smith", "Rogers"),
-        intCol("DeptID", 31, 33, 33, 34, 34, NULL_INT),
-        stringCol("Telephone", "(347) 555-0123", "(917) 555-0198", "(212) 555-0167", "(952) 555-0110", None, None)
-    )
-
-right = newTable(
-        intCol("DeptID", 31, 33, 34, 35),
-        stringCol("DeptName", "Sales", "Engineering", "Clerical", "Marketing"),
-        stringCol("Telephone", "(646) 555-0134", "(646) 555-0178", "(646) 555-0159", "(212) 555-0111")
-    )
-
-t = left.join(right, "DeptID", "DeptName,DeptTelephone=Telephone")
-```
-
-![alt_text](docs/images/ide_first_query.png "Deephaven IDE First Query")
```

Comment:
This is an example of material to get new users to consider the product -- which should not have been deleted.